gulpfile.js

```diff
@@ -107,6 +107,39 @@ var paths = {
     }, {
       src: './node_modules/jquery/dist/jquery.min.js',
       dest: 'js/'
+    }, {
+      src: './node_modules/twemoji-emojis/vendor/svg/1f5c3.svg',
```

Comment:
Would copying all svg too large? `1f5c3` is not really self explainning ;)

Reply:
Ohhhhh... I've just seen `@emoji-map` :/  That seems a bit complex to maintain, no?

Reply:
> Would copying all svg too large? 

There are 3689 files. 8,6Mo. 
Around 1Mo compressed.

> `1f5c3` is not really self explainning ;)

This certainly has to do with the [Emoji 14.0 spec](https://www.unicode.org/reports/tr51/tr51-21.html) which the project adheres to and allows different sets to be used.
Fomantic's documentation only cites 2 sets of emojis. 

I suppose that any other set respecting the specs can be used in place easily. Ex: [Openmoji](https://openmoji.org/)

I can create an `emojis` named task instead of simply adding them to `extras`.
Or if you judge it acceptable, include the whole set of emojis.

> Ohhhhh... I've just seen `@emoji-map` :/ That seems a bit complex to maintain, no?

Noooo :wink: 

This is just an override of the complete list maintained in fomantic : https://github.com/fomantic/Fomantic-UI/blob/develop/src/themes/default/elements/emoji.variables

Galette's theme is based on default. So overrinding this variable prevents from adding full css rules in galette's theme ; only the ones mapped here :grin:

Reply:
> > Would copying all svg too large?
> 
> There are 3689 files. 8,6Mo. Around 1Mo compressed.

OK for me to copy the whole set, so we do not forget anything in further updates.

> > Ohhhhh... I've just seen `@emoji-map` :/ That seems a bit complex to maintain, no?
> 
> Noooo wink
> 
> This is just an override of the complete list maintained in fomantic : https://github.com/fomantic/Fomantic-UI/blob/develop/src/themes/default/elements/emoji.variables
> 
> Galette's theme is based on default. So overrinding this variable prevents from adding full css rules in galette's theme ; only the ones mapped here grin

Ohhhhh... OK; fine for me then ;)